kpi/deployment_backends/mixin.py

```diff
@@ -62,14 +62,14 @@ def has_deployment(self):
     def set_deployment(self, deployment: BaseDeploymentBackend):
         setattr(self, '__deployment_backend', deployment)
 
-    def _mark_latest_version_as_deployed(self, save: bool = False):
+    def _mark_latest_version_as_deployed(self, save: bool = True):
```

Comment:
I could only find this used in one other place: `sync_kobocat_xforms`. It already had `save=True`, so I left it alone.